test/functional/tests/lazy_writes/test_lazy_writes_clean.py

```diff
@@ -0,0 +1,343 @@
+
+    with TestRun.step("Unmount and remove core."):
+        core.unmount()  # the first time unmount from custom mount point
+        core.unmount()  # the second time unmount from xfs filesystem's automatic mount point
+        core.remove_core()
+
+    with TestRun.step("Stop cache."):
+        cache.stop()
+
+    with TestRun.step("Mount core device."):
+        core_part.mount(mnt_point)
+
+    with TestRun.step("Read data from the core device."):
+        test_file_2 = File.create_file("/tmp/test_file_2")
+        dd = Dd().output(test_file_2.full_path) \
+            .input(test_file_1.full_path) \
+            .block_size(bs) \
+            .count(int(test_file_1.size / bs)) \
+            .oflag("direct")
+        dd.run()
+        test_file_2.refresh_item()
+        sync()
+
+    with TestRun.step("Compare md5 sum of test files."):
+        if test_file_md5sum_main != test_file_2.md5sum():
+            TestRun.LOGGER.error("Md5 sums should be equal.")
+        if test_file_1.md5sum() != test_file_2.md5sum():
+            TestRun.LOGGER.error("Md5 sums should be equal.")
+
+    with TestRun.step("Delete test files."):
+        test_file_main.remove(True)
+        test_file_1.remove(True)
+        test_file_2.remove(True)
+
+    with TestRun.step("Unmount core device."):
+        core_part.unmount()
+        remove(mnt_point, True, True, True)
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_clean_remove_core_without_fs(cache_mode):
+    """
+        title: Test of the ability to remove core from cache in lazy-write modes without filesystem.
+        description: |
+          Test if OpenCAS removes core without filesystem in modes with lazy writes
+          without data loss.
+        pass_criteria:
+          - Core removing works properly.
+          - Writes to exported object and core device during OpenCAS's work are equal
+          - Data on core device is correct after core is removed.
+    """
+    with TestRun.step("Prepare devices for cache and core."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(256, Unit.MebiByte)])
+        cache_part = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_dev.create_partitions([Size(512, Unit.MebiByte)])
+        core_part = core_dev.partitions[0]
+        Udev.disable()
+
+    with TestRun.step(f"Start cache in {cache_mode} mode."):
+        cache = casadm.start_cache(cache_part, cache_mode)
+
+    with TestRun.step("Add core to cache."):
+        core = cache.add_core(core_part)
+        # get IO stats independent from test - adding core creates metadata on core device
+        core_disk_writes_diff = IOstatBasic.get_iostat_list([core_part])[0].total_writes.value
```

Comment:
As I wrote before - diff means the difference between some values. I know I wrote "unless you treat 0 as the value for differentiation", and that left an option for leaving this name as is, but I'd prefer renamed variable.